tools/dev/k3d/Makefile

```diff
@@ -66,11 +58,20 @@ secrets: secrets/grafana.jwt secrets/gel.jwt
 
 secrets/grafana.jwt:
 	mkdir -p secrets/
-	op document get "loki/grafana.jwt" --output=$(CURDIR)/secrets/grafana.jwt
+	op document get "loki/grafana.jwt" --output=$(CURDIR)/secrets/grafana.jwt || rm -f $(CURDIR)/secrets/grafana.jwt
+	@if ! test -s "$(CURDIR)/secrets/grafana.jwt"; then \
+		echo "Failed to fetch Grafana license from 1Password, are you logged in?"; \
+		rm -f $(CURDIR)/secrets/grafana.jwt; \
```

Comment:
This instruction is not needed, right? Because the file doesn't exist

Reply:
Ah, forget it! The file can exist, but be empty. In that case, we should also add an `rm -f` to the `secrets/gel.jwt` condition.

Reply:
Yup, that's the thought :smile: